share/spice/piratetalk/piratetalk.css

```diff
@@ -0,0 +1,55 @@
+.brainy__quote--container {
+    overflow: hidden;
+    margin-left: -2.8em;
+}
+
+.brainy__quote {
```

Comment:
I'm guessing you just copy-pasted this? We need to update all of this to reflect new Spice.